### PR TITLE
parameterize kritis-validation-hook-* names in int tests

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by priyawadhwa** [comment]
You might need to add the new names for the webhooks to the CSR cert [here](https://github.com/grafeas/kritis/blob/master/helm-hooks/preinstall/preinstall.go#L60)

**Comment by aaron-prindle** [comment]
yea fixing that rn

**Comment by aaron-prindle** [comment]
preinstall is templated now

File: helm-hooks/preinstall/preinstall.go [hunk 1]
```diff
@@ -49,24 +61,33 @@ func deleteExistingObjects() {
...
 "hosts": [
-    "kritis-validation-hook",
-    "kritis-validation-hook.kube-system",
-    "kritis-validation-hook.%s",
-    "kritis-validation-hook.%s.svc",
-    "kritis-validation-hook-deployments",
-    "kritis-validation-hook-deployments.kube-system",
-    "kritis-validation-hook-deployments.%s",
-    "kritis-validation-hook-deployments.%s.svc"
+    "{{ .ServiceName }}",
```
Comment (dlorenc) [inline helm-hooks/preinstall/preinstall.go]: nit: pull this out to a constant

Comment (dlorenc) [reply helm-hooks/preinstall/preinstall.go]: And you can initialize it with "MustParse"

Comment (aaron-prindle) [reply helm-hooks/preinstall/preinstall.go]: I'm going to merge this now to get the parallel tests for other users and do this in another PR, what is MustParse in this context?  I don't see it in text/template

**Pull request merged**